app/controllers/org_admin/templates_controller.rb

```diff
@@ -39,11 +39,29 @@ def index
       }
     end
     
+    # GET /org_admin/templates/[:id]
+    def show
+      template = Template.includes(:org, :phases).find(params[:id])
```

Comment:
I would get minimum needed for authorize and after pull from the database all the rest of things that are needed